sql/catalyst/src/main/scala/org/apache/spark/sql/types/UserDefinedType.scala

```diff
@@ -50,11 +50,8 @@ abstract class UserDefinedType[UserType] extends DataType with Serializable {
 
   /**
    * Convert the user type to a SQL datum
-   *
```

Comment:
It might also be a good idea to update the documentation. In this case, what exactly is a SQL datum?